members/TXL000214.yaml

```diff
@@ -45,6 +45,11 @@ contact_form:
           selector: "textarea[name='message']"
           value: $MESSAGE
           required: true
+    - javascript:
+        - name: subject truncation
+          selectors: [ "input[name='subject']"]
+          commands: [ "elements[0].value = elements[0].value.length > 44 ? elements[0].value.substring(0,41) + '...' : elements[0].value" ]
```

Comment:
50?

Reply:
It's 44 chars for TX Reps (targets with URLs like this `http://www.house.state.tx.us...`): https://gyazo.com/9398a673439e6b66316060df29d8b651

And 50 chars for TX Sens (targets with URLs like this `http://www.senate.state.tx...`): https://gyazo.com/366825fd59b54ed6af8a5865e010af5c

Reply:
ah fun